### DISCUSSION TITLE
Can I make my own `winget` for Windows 7?

### DISCUSSION THREAD

**Discussion by pybash1** [microsoft/winget-pkgs#18090] [Ideas]
Do I really even need a description I think the title is enough! Anyways still can I make my own version of `winget-cli` for Windows 7?

**Comment by ItzLevvie** [comment]


**Reply by pybash1** [reply]
Ok, Thanks solves my query! But could Microsoft build it then?

**Reply by pybash1** [reply]
I know I was just kidding! Why would Microsoft add support for an end-of-life product and when a new version of Windows is about to launch!

**Reply by Spahatticat** [reply]
# 😄
